setup.py

```diff
@@ -874,17 +864,16 @@ def _postinstall(self):
             raise RuntimeError(f"Can't find '{filename}'")
         # As of setuptools>=74.0.0, we no longer need to
         # be concerned about distutils calling win32api
-        subprocess.Popen(
-            [
+        subprocess.check_call(
+            (
                 sys.executable,
                 filename,
                 "-install",
                 "-destination",
                 self.install_lib,
-                "-quiet",
```

Comment:
At first I did
```suggestion
                *([] if self.verbose else ["-quiet"]),
```
but realized pip with `-v` isn't printing any of this anyway.